common/modifiers/hmo_hardcoded.txt

```diff
@@ -236,17 +237,24 @@ is_republic = {
 
 negative_centralization = {
```

Comment:
need to test these. i thought they were standalone modifies but i think it's for each point of centralization, so need to scale these values accordingly